src/server/hset_family.cc

```diff
@@ -171,6 +172,7 @@ OpStatus OpIncrBy(const OpArgs& op_args, string_view key, string_view field, Inc
     if (pv.ObjType() != OBJ_HASH)
       return OpStatus::WRONG_TYPE;
 
+    op_args.shard->search_indices()->RemoveDoc(key, op_args.db_cntx, it->second);
```

Comment:
can we avoid paying penalty if search is not used?

Reply:
What do you mean by penalty? The cost of a function call / pointer access? I bet its nanoseconds

The function itself won't do anything, because the doc indices array will be empty

Reply:
Ok, I've looked at another `RemoveDoc` function.